packages/signature-controller/src/SignatureController.ts

```diff
@@ -250,6 +251,27 @@ export class SignatureController extends BaseControllerV2<
     return this.#typedMessageManager.getUnapprovedMessagesCount();
   }
 
+  /**
+   * Retrieves an array of all messages from TypedMessages and PersonalMessages.
+   *
+   * @returns The array of all messages from their proxies.
+   */
+  get allMessages(): unknown[] {
```

Comment:
Would it be cleaner to just call this property `messages` as `all` would be implied as it's the default?